package.json

```diff
@@ -12,12 +12,12 @@
   },
   "scripts": {
     "start": "gulp watch",
-    "lint": "gulp lint",
+    "lint": "yarn format && gulp lint",
```

Comment:
this will stop travis failing on lint errors

Reply:
I'll remove this. @bameyrick just a heads up in case you thought it would go in with this PR.